net-mgmt/pfSense-pkg-nrpe/files/usr/local/pkg/nrpe.inc

```diff
@@ -247,7 +247,7 @@ function nrpe_custom_php_validation_command($post, &$input_errors) {
 		}
 	}
 	foreach (explode(",", $post['allowed_hosts']) as $host) {
-		if (!empty($host) && !is_ipaddr($host)) {
+		if (!empty($host) && !is_ipaddr($host) && !is_ipaddrv6($host)) {
```

Comment:
`is_ipaddr()` matches both IPv4 and IPv6, so this change would not alter any behavior.

```
pfSense shell: $host = '10.0.0.1';
pfSense shell: var_dump( (!empty($host) && !is_ipaddr($host)) );
pfSense shell: exec
bool(false)
pfSense shell: var_dump( (!empty($host) && !is_ipaddr($host) && !is_ipaddrv6($host)) );
pfSense shell: exec
bool(false)
pfSense shell: $host = '2001:db8::1';
pfSense shell: var_dump( (!empty($host) && !is_ipaddr($host)) );
pfSense shell: exec
bool(false)
pfSense shell: var_dump( (!empty($host) && !is_ipaddr($host) && !is_ipaddrv6($host)) );
pfSense shell: exec
bool(false)
```

Reply:
in this case we don't need this PR at all